pkg/winupgrade/upgrade.go

```diff
@@ -295,14 +295,52 @@ func verifyImagesSharePathPrefix(first, second string) error {
 }
 
 func verifyPodImageWithHostPathVolume(cs kubernetes.Interface, nodeName string, hostPath string) error {
-	// Get pod list for all pods on this node.
-	list, err := cs.CoreV1().Pods("").List(context.TODO(), metav1.ListOptions{
+	// Get pod list for calico-system pods on this node.
+	list, err := cs.CoreV1().Pods("calico-system").List(context.TODO(), metav1.ListOptions{
 		FieldSelector: "spec.nodeName=" + nodeName,
```

Comment:
I don't think it is correct that we use a nodeName selector since we are on a Windows node which has no calico-node pod.